admin/preprint_providers/views.py

```diff
@@ -159,6 +169,41 @@ def get_success_url(self, *args, **kwargs):
         return reverse_lazy('preprint_providers:detail', kwargs={'preprint_provider_id': self.kwargs.get('preprint_provider_id')})
 
 
+class ProcessCustomTaxonomy(PermissionRequiredMixin, View):
+
+    permission_required = 'osf.view_preprintprovider'
```

Comment:
Shouldn't this be `osf.change_preprintprovider`?

Reply:
Yes, yes it should!